test/zod/test_zod.ts

```
@@ -38,3 +38,21 @@
   assert.deepStrictEqual(expected, data);
 });
+
+test("test zod image path", async () => {
+  const data = mediaSourceSchema.safeParse({kind: "path", path: ""});
+  assert(!data.success);
+  const data2 = mediaSourceSchema.safeParse({kind: "path", path: "1"});
+  assert(data2.success);
+});
+
+test("test zod image url", async () => {
+  const data = mediaSourceSchema.safeParse({kind: "url", url: ""});
```

ESLint

[error] Replace `kind:·"url",·url:·""` with `·kind:·"url",·url:·""·`

(prettier/prettier)